.github/workflows/gh-page.yml

```diff
@@ -0,0 +1,35 @@
+name: main
+
+on:
+  push:
+    branches:
+      - "development"
+      - "add-obsolete-action-command"
```

Comment:
This line can be removed 👀 (It might be for testing.)

Reply:
Yes It's just for testing so it will be removed before merge branch